tutorials/vpc-private-gateway-autoconfiguration/index.mdx

```diff
@@ -0,0 +1,202 @@
+
+3. Click **VPC** in the **Network** section. The list of your Private Networks displays.
+
+4. Click <icon name="more" /> > **More info** to display the details of the Private Network.
+
+5. Click the **Attached Ressources** tab to display a list of resources attached to your Private Network.
+
+7. Select your new Instance in the drop down list and click **Add Instance** to attach it to the Private Network:
+
+  <Lightbox src="scaleway_vpc_add_instance.png" alt="" />
+
+  The new Instance displays in the list of devices connected to the Private Network.
+
+## Configuring port forwarding 
+
+Port forwarding allows to configure a mapping of ports on the IP address of the Public Gateway to IP addresses and ports inside the Private Network. This allows to access services on Virtual Instances in a Private Network without need for a flexible public IP on them. 
+
+1. Click **VPC** in the **Network** section of the side menu in the Scaleway console. The VPC configuration displays.
+
+2. Click the **Public Gateways** tab. The list of your Public Gateways displays. 
+
+3. Click <Icon name="more" /> > **More Info** to display the Public Gateway information. 
+
+4. Click the **DHCP** tab. Then click **View DHCP Status** to display the list of devices autoconfigured by the DHCP server: 
+
+  <Lightbox src="scaleway_vpc_dhcp_status.png" />
+
+  <Message type="tip">
+
+  You notice in the screenshot above, that both entries are **dynamic** types. This means the DHCP server dynamically assigns an IP address to each device. The assigned IP address may change and the port forwarding will stop working. To avoid this, configure a **static** lease. 
+
+  </Message>
+
+5. Copy the **MAC address** of the device you want configure. Then click **Configure DHCP**.
+
+6. Click **Edit** > **Add static association** and enter the MAC address and the private IP address for the Instance. Confirm by clicking <Icon name="validate" />
+
+  <Lightbox src="scaleway_vpc_static_dhcp.png" />
+
+7. Click the **NAT** tab to display the NAT configuration. 
+
+8. Click **+ Add Static NAT** to conigure a new static NAT rule. In this example, we expose the internal port `22` of the Instance `192.168.0.2` to the external port `2202` of the Public Gateway. Confirm by clicking <Icon name="validate" />:
+
+  <Lightbox src="scaleway_vpc_static_nat.png" />
+
+  <Message type="tip">
+
+  You can select to forward either:
+    * TCP traffic
+    * UDP traffic
+  or both. 
+
+  </Message>
+
+You can now connect to your Instance using the IP address of the Public Gateway: 
+
+```
+ssh root@<public_gateway_ip> -p 2202
+```
+
+You are now logged into the first Instance using the IP address of the Public Gateway: 
+
+```
+Welcome to Ubuntu 20.04.3 LTS (GNU/Linux 5.4.0-81-generic x86_64)
+
+ * Documentation:  https://help.ubuntu.com
+ * Management:     https://landscape.canonical.com
+ * Support:        https://ubuntu.com/advantage
+
+root@my-first-instance:~#
+```
```

Comment:
Add something like:

Congratulations, you have completed your first VPC configuration. To move forward, see
- link to PN doc
- link to Public Gateway getting started
- link to Public Gateway advanced